src/test/java/org/folio/rest/impl/AccountsActionChecksAPITests.java

```diff
@@ -44,6 +53,14 @@ public void checkWaiveAmountShouldNotBeAllowedWithExceededAmount() {
     actionCheckAmountShouldNotBeAllowedWithExceededAmount(accountsCheckWaiveClient);
   }
 
+  @Test
+  public void checkTransferAmountShouldNotBeAllowedWithExceededAmount() {
+    Account accountToPost = postAccount();
+    ResourceClient accountsCheckTransferClient =
+      accountsCheckTransferClient(accountToPost.getId());
```

Comment:
Can we move this to @Before or @BeforeClass?

Reply:
moved